internal/improve/files.go

```
@@ -0,0 +1,48 @@
+	})
+}
+
+func copyFile(src, dst string, mode os.FileMode) error {
+	if err := os.MkdirAll(filepath.Dir(dst), 0o755); err != nil {
+		return err
+	}
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `in.Close` is not checked

(errcheck)